flink-core/src/main/java/org/apache/flink/configuration/AkkaOptions.java

```diff
@@ -28,31 +28,143 @@
 @PublicEvolving
 public class AkkaOptions {
 
+	public static String DEFAULT_AKKA_TRANSPORT_HEARTBEAT_INTERVAL = "1000 s";
+
+	public static String DEFAULT_AKKA_TCP_TIMEOUT = "20 s";
+
+	public static String DEFAULT_AKKA_WATCH_HEARTBEAT_INTERVAL = "10 s";
```

Comment:
Thank you for your suggestion, and I was also bothered by the `DEFAULT_AKKA_*` fields while the default value is used. `ConfigOption#defaultValue()` sounds good.